app-nexus-network/app-nexus-network-htb.js

```diff
@@ -75,7 +75,7 @@ function AppNexusNetworkHtb(configs) {
      */
     var __baseUrl;
 
-    var __version = '2.4.0';
+    var __version = '2.5.0';
```

Comment:
We see that you have included the latest partner update changes (allowSmallerSizes, usePaymentRule).

As we are on a tight time constraints, we would like to only have the CCPA changes in this PR.

Can you please branch off from master and apply only the CCPA changes.

Thanks!

Reply:
Please scratch this request. We can work with this as is.